packages/moby-buildx/go_version.go

```diff
@@ -6,5 +6,5 @@ import (
 )
 
 func GoVersion(_ *archive.Spec) string {
-	return goversion.OneTwentyThree
+	return goversion.OneTwentyFour
```

Comment:
tbh, never seen something like this before

Reply:
We needed some projects to stay on an older go for some time but some required new go (really... compose *always* bumps to new go immediately, its frustrating).
Right now everything is good on go1.24.
At some point I'm sure something (*cough* compose *cough*) will force a bump to go1.25 and we'll need to have a split again.